src/pages/PlaylistPage.js

```diff
@@ -79,7 +79,7 @@ const mapStateToProps = (state, {params}) => ({
   lessonsByLevel: getLessonsByLevel(state, params.course),
   coursePlaylists: getPlaylists(state, params.course),
   t: getTranslator(state),
-  showPlaylists: state.showPlaylists,
+  showPlaylists: state.showPlaylists && Object.keys(getPlaylists(state, params.course)).length > 0,
```

Comment:
Hvis du gjør som jeg foreslår over, så bør det stå
```
  showPlaylists: state.showPlaylists && getShowRadiobuttons(state, params.course),
```
for ellers blir det funky hvis du velger engelsk som hovedspråk (forsøk selv før du endrer noe!)